Hotel/forms.py

```diff
@@ -9,4 +9,17 @@ class RegisterForm(UserCreationForm):
                              widget=forms.TextInput)
     class Meta:
      model=User
-     fields=('username','email','password1','password2')
\ No newline at end of file
+     fields=('username','email','password1','password2')
+class BookingForm(forms.Form):
+    room_type = forms.ModelChoiceField(queryset=RoomType.objects.all())
+    check_in = forms.DateField()
+    check_out = forms.DateField()
+
+class RoomForm(forms.ModelForm):
+    class Meta:
+        model = Rooms
+        fields = ['number', 'room_type', 'price','capacity', 'status']
+class RoomaddForm(forms.ModelForm):

```

Comment:
Why this one ?

Reply:
roodaddform is for adding of new room and roomform is for editing of already existing room . roomaddform contains all attributes .whereas in roomform hotel attribute is not present as it is not  to change hotel associated with the room as edit feature of room